generate_settings.py

```diff
@@ -203,7 +203,11 @@ def main(argv):
     series = get_series(boot_hart, bitness)
     intr_wait_cycle = 0
     if series is not None:
-        intr_wait_cycle = 5000 if "8" in series else 0
```

Comment:
why not just fix the software-interrupt test to just always wait for 'a while'. I don't see any benefit to having a small value used on some platforms for that test.